teamA/crud-express/heeyeon/routes/index.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const router = express.Router();
+
+const userRouter = require('./userRouter');
+router.use('/users', userRouter.router);
+
+const postRouter = require('./postRouter');
+router.use('/posts', postRouter.router);
+
+const likesRouter = require('./likesRouter');
+router.use('/likes', likesRouter.router);

```

Comment:
희연님 해당 코드는 가독성을 위해서 아래와 같이 수정해 주세요 !
```suggestion
const userRouter = require('./userRouter');
const postRouter = require('./postRouter');
const likesRouter = require('./likesRouter');

router.use('/users', userRouter.router);
router.use('/posts', postRouter.router);
router.use('/likes', likesRouter.router);

```